views/email_template/UserInviteSelf.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+use humhub\modules\ui\mail\DefaultMailStyle;
+use yii\helpers\Url;
+use yii\helpers\Html;
+
```

Comment:
can you show a preview screenshot of this email please?

Reply:
![image](https://user-images.githubusercontent.com/103625795/172352898-a107ef0f-0a26-4227-bc43-b4a54cbcd3ec.png)